src/unstract/adapters/llm/vertex_ai/src/static/json_schema.json

```diff
@@ -39,6 +39,14 @@
       "title": "Max Retries",
       "default": 5,
       "description": "Maximum number of retries to attempt when a request fails."
+    },
+    "max_tokens": {
+      "type": "number",
+      "minimum": 0,
+      "multipleOf": 1,
+      "title": "Max output tokens",
+      "default": 2048,
```

Comment:
Setting a default here because when not set, llama-index defaults to 512 which is very less.